Solutions/PDNS Block Data Connector/DataConnectors/PDNSBlockDataConnector_API_FunctionApp.json

```diff
@@ -0,0 +1,122 @@
+{
+	"id": "PDNSBlockDataConnector",
+	"title": "PDNS Block Data Connector",
+	"publisher": "Nominet",
+	"descriptionMarkdown": "This application enables you to ingest your PDNS block data into your SIEM tool",
+    "graphQueries": [
+		{
+			"metricName": "Total data received",
+			"legend": "PDNSBlockData_CL",

```

Comment:
Hi @TedWhi please provide sample data related to 'PDNSBlockData_CL' to test connector queries, thanks!

Reply:
Have just pushed up a file with sample data :)